app/controllers/patients_controller.rb

```diff
@@ -31,6 +31,10 @@ def create
 
   def pledge
     @patient = Patient.find params[:patient_id]
+    if @patient.update_attributes params[:pledge_sent]
```

Comment:
I totally had to look this up and see how it was being used -- it's being used to populate the modal that pops up when you click the big purple `Submit Pledge` button in the sidebar menu. Probably we should have this bump be a condition of `update`, can we shift that down there into the block that happens if the update completes?